packages/file_selector/file_selector_android/android/src/main/java/io/flutter/plugins/file_selector/FileSelectorDelegate.java

```diff
@@ -38,12 +48,24 @@
     implements PluginRegistry.ActivityResultListener,
         PluginRegistry.RequestPermissionsResultListener {
   @VisibleForTesting static final int REQUEST_CODE_GET_DIRECTORY_PATH = 2342;
+  @VisibleForTesting static final int REQUEST_CODE_OPEN_FILE = 2343;
 
   /** Constants for key types in the dart invoke methods */
-  @VisibleForTesting static final String _confirmButtonText = "confirmButtonText";
-
-  @VisibleForTesting static final String _initialDirectory = "initialDirectory";
+  @VisibleForTesting
+  static final String _confirmButtonText = "confirmButtonText";
+  @VisibleForTesting
+  static final String _initialDirectory = "initialDirectory";
+  @VisibleForTesting
+  static final String _acceptedTypeGroups = "acceptedTypeGroups";
+  @VisibleForTesting
+  static final String _multiple = "multiple";
+  @VisibleForTesting
+  static String cacheFolder = "file_selector";
+  @VisibleForTesting
+  public PathUtils pathUtils = new PathUtils();
```

Comment:
we don't need to create an instance to `pathUtils` here since we can use the static methods from path utils as PathUtils.{method}. I'll add a change request in those below